userguide/content/en/docs/adding-content/diagrams-and-formulae/index.md

```diff
@@ -342,48 +342,63 @@ With Mermaid support enabled in Docsy, you can include the text definition of a
 
 The great advantage of this is anyone who can edit the page can now edit the diagram - no more hunting for the original tools and version to make a new edit.
 
-For example, the following defines a simple flowchart:
+For example, the following defines a sequence diagram:
 
 ````
 ```mermaid
-graph TD
-  Start --> Need{"Hugo version >= 0.93.0"}
-  Need -- No --> Off["Set params.mermaid.enable = true"]
-  Off --> Author
-  Need -- Yes --> Author[Insert mermaid codeblock]
-```
+sequenceDiagram
+    autonumber
+    Docsy user->>Discussion board: Ask question
+    Discussion board->>Community member: read question
+    loop Different strategies
+    Community member->>Test instance: Investigate issue raised
+    end
+    Note right of Community member: After hours of investigation:
+    Test instance-->>Community member: Come up with solution
+    Community member-->>Discussion board: Propose solution
+    Discussion board-->>Docsy user: check proposed solution
+    Docsy user->>Discussion board: Mark question as resolved
+    Docsy user->>Docsy user: Being happy```
```

Comment:
~~~suggestion
    Docsy user->>Docsy user: Being happy
```
~~~

Reply:
Thanks for spotting this, changed as requested.